R/temporalBehaviour.R

```diff
@@ -425,6 +425,11 @@ temporalBehaviour <- function(sts,
                               empiricalRMW = FALSE,
                               tempRes = 1,
                               verbose = 1) {
+  
+  if(is.null(windThreshold)){
+    windThreshold = sts@scale
+  }
+  
```

Comment:
Change the default value of `windThreshold` in the call of `temporalBehaviour`

Reply:
Yes, done, I forgot